lte/gateway/python/integ_tests/s1aptests/test_sctp_shutdown_while_mme_is_stopped.py

```diff
@@ -68,19 +69,14 @@ def test_sctp_shutdown_while_mme_is_stopped(self):
         self._s1ap_wrapper.magmad_util.print_redis_state()
 
         print("Starting MME service and waiting for 30 seconds")
-        self._s1ap_wrapper.magmad_util.exec_command(
-            "sudo service magma@mobilityd start",
-        )
-        self._s1ap_wrapper.magmad_util.exec_command(
-            "sudo service magma@pipelined start",
-        )
-        self._s1ap_wrapper.magmad_util.exec_command(
-            "sudo service magma@sessiond start",
-        )
-        self._s1ap_wrapper.magmad_util.exec_command(
-            "sudo service magma@mme start",
+        self._s1ap_wrapper.magmad_util.enable_service('mobilityd')
+        self._s1ap_wrapper.magmad_util.enable_service('pipelined')
+        self._s1ap_wrapper.magmad_util.enable_service('sessiond')
+        self._s1ap_wrapper.magmad_util.enable_service('mme')
+        self._s1ap_wrapper.magmad_util.wait_for_restart_to_finish(
```

Comment:
Please remove service list. The prototype of this function expects only one parameter, i.e., wait time:

```
    @staticmethod
    def wait_for_restart_to_finish(wait_time):
```

Reply:
Thank you, I missed that. The changes to the wait function are not merged yet.